Checking: mail.mtn.com
Domain: mail.mtn.com : HTTP Error: HTTPConnectionPool(host='mail.mtn.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f59251412b0>, 'Connection to mail.mtn.com timed out. (connect timeout=5)'))
Domain: mail.mtn.com : Ping: Unreachable
Domain: mail.mtn.com : Telnet: Port 80 is closed (Timeout)
--------------------------------------------------
Checking: za-jh-bums-01.sig.uc.mtn.com
Domain: za-jh-bums-01.sig.uc.mtn.com : HTTP Error: HTTPConnectionPool(host='za-jh-bums-01.sig.uc.mtn.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f59251419d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Domain: za-jh-bums-01.sig.uc.mtn.com : Ping: Unreachable
Domain: za-jh-bums-01.sig.uc.mtn.com : Telnet: Error - [Errno 111] Connection refused
--------------------------------------------------
Checking: help.staging-my.mtn.com
Domain: help.staging-my.mtn.com : HTTP Error: HTTPConnectionPool(host='help.staging-my.mtn.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5925141f10>, 'Connection to help.staging-my.mtn.com timed out. (connect timeout=5)'))
Domain: help.staging-my.mtn.com : Ping: Reachable
Domain: help.staging-my.mtn.com : Telnet: Port 80 is closed (Timeout)
--------------------------------------------------
Checking: ulearn.mtn.com
Domain: ulearn.mtn.com : Status Code: 200 OK
Domain: ulearn.mtn.com : Ping: Reachable
Domain: ulearn.mtn.com : Telnet: Port 80 is open
--------------------------------------------------
Checking: vctms.mtn.com
Domain: vctms.mtn.com : HTTP Error: HTTPConnectionPool(host='vctms.mtn.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f5925142390>, 'Connection to vctms.mtn.com timed out. (connect timeout=5)'))
